sdk/keyvault/Azure.Security.KeyVault.Certificates/samples/Sample2_GetCertificates.md

```diff
@@ -0,0 +1,106 @@
+# Listing certificates, certificate versions and deleted certificates
+
+This sample demonstrates how to list certificates and versions of given certificates, and list deleted certificates in a soft delete-enabled key vault.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](../README.md) for links and instructions.
```

Comment:
This is repeated from another file.  Consider putting the shared content into an include and referencing here.

Reply:
Includes are not processed for markdown viewed on GitHub.